File: 0125-valid-palindrome/0125-valid-palindrome.py
class Solution:
    def isPalindrome(self, s: str) -> bool:
        p = ""

        for char in s:
            if char.isalnum():
                p += char
        L , R = 0, len(p) - 1
        p = p.lower()
        while L < R:
            if p[L] == p[R]:
                L += 1
                R -= 1
            else:
                return False

        return True
 

File: 0125-valid-palindrome/0125-valid-palindrome.cpp
class Solution {
public:
    bool isPalindrome(string s) {
        string real = "";
        for(int i = 0; i < s.length(); i++){
            if(isalnum(s[i])){
                real += tolower(s[i]);
            }
        }

    int counter = real.length()-1;

    for(int i = 0; i < real.length(); i++){
        if(real[counter]!=real[i]){
            return false;
        }
        counter--;
    }
    return true;
    }
};